packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -423,7 +423,9 @@ export const Stackbox = () => {
       <div className="px-5 py-6 space-y-6">
         <div className="space-y-2">
           <TitleText weight="bold" className="text-em-med">
-            Stack WETH every
+            {toToken
+              ? `Stack ${toToken?.symbol} every`
+              : "Select a token to receive"}
```

Comment:
The default should be _"Stack every"_. 

```suggestion
              : "Stack every"}
```

Reply:
Even if you don't have a `toToken` selected?

Reply:
yes

Reply:
Let's go with "Stack every". Also it's pretty much impossible to have no token selected in our interface